src/tlo/methods/cervical_cancer.py

```diff
@@ -0,0 +1,1515 @@
+            "selected for xpert this month"
+        ),
+        "ce_biopsy": Property(
+            Types.BOOL,
+            "ce biopsy done"
+        )
+    }
+
+    def read_parameters(self, data_folder):
+        """Setup parameters used by the module, now including disability weights"""
+        # todo: add disability weights to resource file
+
+        # Update parameters from the resourcefile
+        self.load_parameters_from_dataframe(
+            pd.read_excel(Path(self.resourcefilepath) / "ResourceFile_Cervical_Cancer.xlsx",
+                          sheet_name="parameter_values")
+        )
+
+        # todo: specify this odds ratio in parameter file if possible'
+        # Register Symptom that this module will use
+        self.sim.modules['SymptomManager'].register_symptom(
+            Symptom(name='vaginal_bleeding',
+                    odds_ratio_health_seeking_in_adults=2.00)
+        )
+
+# todo: in order to implement screening for cervical cancer creating a dummy symptom - likely there is a better way
+        self.sim.modules['SymptomManager'].register_symptom(
+            Symptom(name='chosen_via_screening_for_cin_cervical_cancer',
+                    odds_ratio_health_seeking_in_adults=100.00)
+        )
+
+        self.sim.modules['SymptomManager'].register_symptom(
+            Symptom(name='chosen_xpert_screening_for_hpv_cervical_cancer',
+                    odds_ratio_health_seeking_in_adults=100.00)
+        )
+
+
+    def initialise_population(self, population):
+        """Set property values for the initial population."""
+        df = population.props  # a shortcut to the data-frame
+        p = self.parameters
+        rng = self.rng
+
+        # defaults
+        df.loc[df.is_alive, "ce_hpv_cc_status"] = "none"
+        df.loc[df.is_alive, "ce_date_diagnosis"] = pd.NaT
+        df.loc[df.is_alive, "ce_date_treatment"] = pd.NaT
+        df.loc[df.is_alive, "ce_stage_at_which_treatment_given"] = "none"
+        df.loc[df.is_alive, "ce_date_palliative_care"] = pd.NaT
+        df.loc[df.is_alive, "ce_date_death"] = pd.NaT
+        df.loc[df.is_alive, "ce_new_stage_this_month"] = False
+        df.loc[df.is_alive, "ce_stage_at_diagnosis"] = "none"
+        df.loc[df.is_alive, "ce_ever_treated"] = False
+        df.loc[df.is_alive, "ce_cc_ever"] = False
+        df.loc[df.is_alive, "ce_xpert_hpv_ever_pos"] = False
+        df.loc[df.is_alive, "ce_via_cin_ever_detected"] = False
+        df.loc[df.is_alive, "ce_date_cryo"] = pd.NaT
+        df.loc[df.is_alive, 'ce_current_cc_diagnosed'] = False
+        df.loc[df.is_alive, "ce_selected_for_via_this_month"] = False
+        df.loc[df.is_alive, "ce_selected_for_xpert_this_month"] = False
+        df.loc[df.is_alive, "ce_biopsy"] = False
+
+
+        # -------------------- ce_hpv_cc_status -----------
+        # this was not assigned here at outset because baseline value of hv_inf was not accessible - it is assigned
+        # st start of main polling event below
```

Comment:
why is baseline value for  `hv_inf ` not accessible yet HIV module is included in the dependencies section and `hv_inf` has been initialised [here in Hiv module](https://github.com/UCL/TLOmodel/blob/416027c84b1e882b10ce0de5dbf2b9339f2d9cb9/src/tlo/methods/hiv.py#L579)? what was the error?